.github/workflows/ci.yml

```diff
@@ -708,6 +706,7 @@ jobs:
           - test_name: lit-ii-batch-test-multiworker
           - test_name: lit-ii-identity-multiworker-test
           - test_name: lit-ii-vc-multiworker-test
+          - test_name: lit-resume-worker
```

Comment:
Failure of this job doesn't fail whole action so we will run it for several days to see whether it still causes the problems.

I've performed more than 10 runs locally and it never failed.